index.js

```diff
@@ -91,7 +92,7 @@ const handleSubmition = (event) => {
   const title = document.querySelector('.title-input').value;
   const author = document.querySelector('.author-input').value;
   const id = generateId();
-  const newBook = { title, author, id };
+  const newBook = new Book({ title, author, id });
   if (bookStore.add(newBook)) {
     displayBook(newBook, bookListElement);
   }

```

Comment:
- [ ] I see that the `handleRemove` function is the function that removes the book layout from the books list layouts. what do think if we add this function to the BookStore? We will keep our code simple and more readable.

Reply:
The `handleRemove` function is an [EventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventListener) that
implements the feature you mentioned but adding it to the `BookStore` class defeats the class single responsibility principle (see PR description).

Reply:
- Yes, `handleRemove` is a function that is used for an eventListener. but it's still handling the BookStore layouts.  Anyway, It's just a suggestion.  I read your PR description again, you are right. Keep going 👍